data/core/units/undead/Corpse_Soulless.cfg

```diff
@@ -27,8 +27,12 @@
 
 # Variant animations for the Soulless
 #define UNIT_BODY_SOULLESS_GRAPHICS BASE_NAME
+#arg IPF
+NOP#endarg
+#arg ARG
+#endarg
```

Comment:
Seems unnecessarily complicated to split this into two arguments with a NOP default instead of just one argument.

Reply:
Sure. I am unhappy with these NOP calls as well. WML macros cannot handle parentheses correctly, that is the reason for that mess. I will revise that and create new sprites with Imagemagick. That would also fix the issue with the IPF filter not showing up in the info bar. Please leave this PR open!

Reply:
> WML macros cannot handle parentheses correctly

Uhh… I'm pretty sure this is not true? If you quote it, there should be no issue.

Reply:
Simplified the macro in 1dd72e7.